extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -138,6 +156,23 @@ class SchemaCompareTester {
 			// TODO : add wait for tasks to complete
 			// script generation might take too long and the 'success' status does not mean that script is created.
 			await this.assertScriptGenerationResult(status, target.serverName, target.databaseName);
+
+			// save to scmp
+			const filepath = path.join(folderPath,`ads_schemaCompare_${now.getTime().toString()}.scmp`);
+			if (!fs.existsSync(folderPath)) {
+				fs.mkdirSync(folderPath);
+			}
+			const saveScmpResult = await schemaCompareService.schemaCompareSaveScmp(source, target, azdata.TaskExecutionMode.execute, null, filepath, [], []);
+			assert(saveScmpResult.success && !saveScmpResult.errorMessage, `Save scmp should succeed. Expected: there should be no error. Actual Error message: "${saveScmpResult.errorMessage}`);
+			assert(fs.existsSync(filepath), `File ${filepath} is expected to be present`);
+
+			// open scmp
+			const openScmpResult = await schemaCompareService.schemaCompareOpenScmp(filepath);
+			assert(openScmpResult.success && !openScmpResult.errorMessage, `Open scmp should succeed. Expected: there should be no error. Actual Error message: "${openScmpResult.errorMessage}`);
+			assert(openScmpResult.sourceEndpointInfo.databaseName === source.databaseName, `Expected: source database to be ${source.databaseName}, Actual: ${openScmpResult.sourceEndpointInfo.databaseName}`);
+			assert(openScmpResult.targetEndpointInfo.databaseName === target.databaseName, `Expected: target database to be ${target.databaseName}, Actual: ${openScmpResult.targetEndpointInfo.databaseName}`);
+
+			fs.unlinkSync(filepath);
```

Comment:
>fs.unlinkSync(filepath); [](start = 3, length = 24)

This should be in a finally so we always clean up the file even if an assert fails the test earlier